### PR TITLE
Add and implement Term() on Message type

### PR DESCRIPTION
**Pull request by joelrebel**
- [events/nats: Add and implement Term() method on event Message type](https://github.com/metal-toolbox/hollow-toolbox/commit/c7aeb3d0ff4ed7afd165ff2db28f5ce47281a466)
- [events/nats: fixes returned error type and sets Name on the consumer.](https://github.com/metal-toolbox/hollow-toolbox/commit/dd62bd7205ebb63f8495130086ee3829a4cc058f)
- [go: update dependencies, closes renovate bot issues](https://github.com/metal-toolbox/hollow-toolbox/commit/e5a2576d94d620a3abe6111b8ecea565cb0a67bc)

### CODE REVIEW

File: events/nats.go [hunk 1]
```diff
@@ -209,6 +209,7 @@ func (n *NatsJetstream) addConsumer() error {
 	// https://pkg.go.dev/github.com/nats-io/nats.go#ConsumerConfig
 	cfg := &nats.ConsumerConfig{
 		Durable:       n.parameters.Consumer.Name,
+		Name:          n.parameters.Consumer.Name,
```
Comment (DoctorVin) [inline events/nats.go:212]: Should `Durable` repeat this data?

**Review by DoctorVin — APPROVED** [review]
I think this is fine, but I'd like to see the removal of `pubsubx.Message` and its like in the near future.

**Comment by DoctorVin** [comment]
Given that this PR is failing due to the use of deprecated types, perhaps we should just rework the message types to FIaaS specific ones now?

**Comment by joelrebel** [comment]
Closing since a recent PR implements Term() , will create a separate PR for the error fix

**Pull request closed without merging**